cypress/e2e/Testing/client-user.cy.js

```diff
@@ -0,0 +1,64 @@
+describe("User tests", () => {
+    it("CAN login and view their profile", () => {
+      let email = "Norofftest@noroff.no";
+      let password = "Norofftest@noroff.no";
+  
+      cy.visit("https://noroff-anders.github.io/Workflow-CA/");
```

Comment:
```suggestion
      cy.visit("/");
```

Cypress has its own webserver which you should be using. This will test static deployed code, not the code in your branch.

Reply:
Updated to sugested souloution